docs/aws/README.md

```diff
@@ -132,29 +144,31 @@ spec:
             requests:
               cpu: 1
 EOF
-kubectl scale deployment inflate --replicas 5
-kubectl logs -f -n karpenter $(kubectl get pods -n karpenter -l karpenter=controller -o name)
 ```
+## Automatic Node Provisioning 
 
-You can see what EC2 instance type was added to your cluster from Karpenter with
-```bash
-kubectl get no -L "node.kubernetes.io/instance-type"
-```
+Scale the previous deployment to 5 replicas, and begin watching the karpenter pod for log events. 
 
-If you scale down the deployment replicas the instance will be terminated after 30 seconds (ttlSeconds).
 ```bash
-kubectl scale deployment inflate --replicas 0
+kubectl scale deployment inflate --replicas 5
+kubectl logs -f -n karpenter $(kubectl get pods -n karpenter -l karpenter=controller -o name)
 ```
 
-Or you can manually delete the node with
+## Automatic Node Deprovisioning
+
+Now, delete the deployment. After 30 seconds (`ttlSecondsAfterEmpty`), karpenter should deprovision the now empty nodes. 
 
-> Karpenter automatically adds a node finalizer to properly cordon and drain nodes before they are terminated.
 ```bash
-kubectl delete node $NODE_NAME
```

Comment:
Why was this example flow deleted?

Reply:
I didn't fully consider the impact of this example. I added a section calling out this behavior.